### PR TITLE
DHFPROD-10119: Matching Step Details Incorrect Column Ordering

### PR DESCRIPTION
**Pull request by amejiaq**
#### Checklist: 
```diff
- Note: do not change the below
```

- ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [n/a] Added Tests
- [x] Ran newly added/edited cypress tests on Firefox locally
  

- ##### Reviewer:

- [x] Reviewed Tests
- [n/a] Added to Release Wiki

### CODE REVIEW

**Review by briantang — CHANGES_REQUESTED** [review]
Can we add a small e2e check here to avoid this in the future since there are so many different types of compare modals? Something just to verify the columns are in the correct order with the proper text labels, i.e. only one "Preview" and two original uri's etc.

For example, in matching.cy.tsx between lines 553 and 566 is a good place to add in a minor check.

**Comment by amejiaq** [comment]
@briantang  Test added!

File: marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/curate/matching.cy.tsx [hunk 1]
```diff
@@ -551,6 +551,23 @@ describe("Matching", () => {
       // To test compare values for matched Uris
       cy.log("**To test compare values for matched Uris**");
       cy.findAllByLabelText("/json/persons/first-name-double-metaphone compareButton").first().scrollIntoView().click();
+      cy.waitForAsyncRequest();
+      // Check  column Order
+      cy.get(".modal-body")
+        .within(() => {
+          cy.get("th").eq(2).within(() => {
```
Comment (briantang) [inline marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/curate/matching.cy.tsx:558]: I think the validated scenario here is flipped. It should be (in pseudo):
th (2) = Current Document: "/json/persons/first-name-double-metaphone1.json"
th (3) = Current Document: "/json/persons/first-name-double-metaphone2.json"
th (4) = Preview:

**Review by briantang — CHANGES_REQUESTED** [review]

File: marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/curate/matching.cy.tsx (hunk 1 above)
Comment (amejiaq) [reply marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/curate/matching.cy.tsx:558]: Fixed ✅

**Review by briantang — APPROVED** [review]
Looks good!

**Review by littlecherro — APPROVED** [review]

**Pull request merged**